app/validators/questions/mutually_exclusive_validator.py

```diff
@@ -6,14 +7,28 @@ class MutuallyExclusiveQuestionValidator(QuestionValidator):
     MUTUALLY_EXCLUSIVE_CONTAINS_MANDATORY = (
         "MutuallyExclusive question type cannot contain mandatory answers."
     )
-    NON_CHECKBOX_ANSWER = "Question is not of type Checkbox."
+    NON_CHECKBOX_RADIO_ANSWER = "Question is not of type Checkbox or Radio."
```

Comment:
This might be better as `INVALID_EXCLUSIVE_ANSWER`. I think the error text should mention mutually exclusive as it doesn't make sense without it.